logserver.go

```diff
@@ -104,8 +104,10 @@ func serveFile(w http.ResponseWriter, r *http.Request, path string) {
 //
 // Unlike mime.TypeByExtension, the results are limited to a set of types which
 // should be safe to serve to a browser without introducing XSS vulnerabilities.
+// These match file file extensions we allow on upload, plus 'log' which we use
+// for the log files.
```

Comment:
slightly confused what we mean by "log files" here. How do they differ from the files we allow on upload? (Aren't log files uploaded?)

Reply:
Commented